variables.tf

```diff
@@ -0,0 +1,25 @@
+variable "aws_region" {
```

Comment:
This can be a data source - just check where in which region the provider currently is.